apps/entitree-images/pages/api/auth/[...nextauth].ts

```diff
@@ -17,7 +17,7 @@ export default async function auth(req, res) {
       async session({ session, token, user }) {
         const userDoc = await prismaClient.user.findFirst({
           where: {
-            id: user.id,
+            id: parseInt(user.id),
```

Comment:
this should not be coming from a URL, how come casting is needed?

Reply:
because it comes from a string